cluster/validate-cluster.sh

```diff
@@ -170,7 +170,7 @@ while true; do
 done
 
 echo "Validate output:"
-kubectl_retry get cs
+return_value=$(kubectl_retry get cs)
```

Comment:
This is wrong. `return_value` is used in `exit` call, which expects integer, plus you shouldn't overwrite what's currently stored in `return_value`.

IIUC your main goal is to to prevent `errexit` to kill the script when `get cs` fails. If that's the case you should write:
```
echo $(kubectl_retry get cs || true)
```